src/main/java/nextstep/subway/favorite/ui/FavoriteController.java

```diff
@@ -0,0 +1,54 @@
+package nextstep.subway.favorite.ui;
+
+import java.net.URI;
+import java.util.List;
+import nextstep.subway.auth.domain.AuthenticationPrincipal;
+import nextstep.subway.auth.domain.LoginMember;
+import nextstep.subway.common.exception.ControllerExceptionHandler;
+import nextstep.subway.favorite.application.FavoriteService;
+import nextstep.subway.favorite.dto.FavoriteRequest;
+import nextstep.subway.favorite.dto.FavoriteResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/favorites")
+public class FavoriteController extends ControllerExceptionHandler {
+    private final FavoriteService favoriteService;
+
+    public FavoriteController(FavoriteService favoriteService) {
+        this.favoriteService = favoriteService;
+    }
+
+    @PostMapping
+    public ResponseEntity<FavoriteResponse> createFavorite(
+        @AuthenticationPrincipal final LoginMember loginMember,
+        @RequestBody final FavoriteRequest request
+    ) {
+        final FavoriteResponse favorite = favoriteService.createFavorite(loginMember.getId(), request);
+        return ResponseEntity.created(URI.create("/favorites/" + favorite.getId())).body(favorite);
+    }
+
+    @GetMapping
+    public ResponseEntity<List<FavoriteResponse>> findFavorites(
+        @AuthenticationPrincipal final LoginMember loginMember
+    ) {
+        List<FavoriteResponse> favorites = favoriteService.findFavoritesByMemberId(loginMember.getId());
+        return ResponseEntity.ok().body(favorites);
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity deleteFavorite(
```

Comment:
```suggestion
    public ResponseEntity<Void> deleteFavorite(
```

return body 가 없을때도 이런식으로 쓰는게 더 좋아보이더라구요 :)